packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -216,24 +217,52 @@ class GoogleChromeDevice extends ChromiumDevice {
 }
 
 /// The Microsoft Edge browser based on Chromium.
-// This is not currently used, see https://github.com/flutter/flutter/issues/55322
 class MicrosoftEdgeDevice extends ChromiumDevice {
   MicrosoftEdgeDevice({
     @required ChromiumLauncher chromiumLauncher,
     @required Logger logger,
     @required FileSystem fileSystem,
-  }) : super(
+    @required ProcessManager processManager,
+  }) : _processManager = processManager,
+       super(
          name: 'edge',
          chromeLauncher: chromiumLauncher,
          logger: logger,
          fileSystem: fileSystem,
        );
 
+  final ProcessManager _processManager;
+
+  // The first version of Edge with chromium support.
+  static const int _kFirstChromiumEdgeMajorVersion = 79;
+
   @override
   String get name => 'Edge';
 
+  Future<bool> _meetsVersionContraint() async {
+    final String rawVersion = (await sdkNameAndVersion).replaceFirst('Microsoft Edge ', '');
+    final Version version = Version.parse(rawVersion);
+    if (version == null) {
+      return false;
+    }
+    return version.major >= _kFirstChromiumEdgeMajorVersion;
+  }
+
   @override
-  Future<String> get sdkNameAndVersion async => '<?>';
+  Future<String> get sdkNameAndVersion async => _sdkNameAndVersion ??= await _getSdkNameAndVersion();
+  String _sdkNameAndVersion;
+  Future<String> _getSdkNameAndVersion() async {
+    final ProcessResult result = await _processManager.run(<String>[
+      r'reg', 'query', r'HKEY_CURRENT_USER\Software\Microsoft\Edge\BLBeacon', '/v', 'version',
+    ]);
+    if (result.exitCode == 0) {
+      final List<String> parts = (result.stdout as String).split(RegExp(r'\s+'));
+      if (parts.length > 2) {
+        return 'Microsoft Edge ' + parts[parts.length - 2];
+      }
+    }
+    return '?';
```

Comment:
Is this a useful value? Maybe extract into a constant and add docs? If not, how about throwing `ToolExit` instead?

Reply:
If we throw a tool exit, then we would not be able to check for whether edge meets the installed version constraint. Will add a comment